src/main/resources/services/rss/rss.js

```diff
@@ -0,0 +1,37 @@
+const portalLib = require('/lib/xp/portal');
+const contentLib = require('/lib/xp/content');
+const rssXmlLib = require('/lib/rss-xml');
+
+exports.get = function(req) {
+    const params = req.params;
+
+    if (params.method === 'list') {
+        const results = contentLib.query({ contentTypes: [`${app.name}:rss-page`] });
+
+        const body = results.hits.map(hit => {
+            const site = contentLib.getSite({ key: hit._id });
+
+            return { 
+                id: hit._id,
+                path: hit._path.replace(site._path + '/', ''),
+                displayName: hit.displayName,
+            }
+        });
+
+        return { body, contentType: 'application/json' }; 
+    }
+
+    if (params.method === 'xml' && params.id) {
```

Comment:
```suggestion
    }
    else if (params.method === 'xml' && params.id) {
```

Reply:
Not needed since we return on every conditional block and also outside of them.

Reply:
They are logically linked. 
The result is the same, so lets just ignore it.